### ISSUE TITLE
"The Content-Type header included with the request is not allowed" error when assigning roles to users

### ISSUE THREAD

**Issue by jeffshurtliff** [jeffshurtliff/khoros#45]
# Description
When attempting to assign roles to users with the `khoros.roles.assign_roles_to_user()` function, the following error response is getting returned:

```python
{
  'status': 'error', 
  'message': 'The Content-Type header included with the request is not allowed (it should be either application/json or application/xml).', 
  'data': {
    'type': 'error_data', 
    'code': 405, 
    'developer_message': '', 
    'more_info': ''}, 
  'metadata': {}
}
```

# Affected Version
```python
>>> import khoros
>>> khoros.__version__
'4.2.0'
```

**Comment by jeffshurtliff** [comment]
Confirmed that the fix in [v4.2.1rc1](https://pypi.org/project/khoros/4.2.1rc1/) resolves the issue.  
Change will be deployed in GA version **4.2.1** shortly.

**Comment by jeffshurtliff** [comment]
The minor release [v4.2.1] (https://pypi.org/project/khoros/4.2.1/) is now available which resolves the issue.
